internal/definitions/definition_handler_identity_claim.go

```diff
@@ -30,18 +30,18 @@ func (dh *definitionHandlers) handleIdentityClaimBroadcast(ctx context.Context,
 	var claim core.IdentityClaim
 	valid := dh.getSystemBroadcastPayload(ctx, msg, data, &claim)
 	if !valid {
-		return HandlerResult{Action: ActionReject}, nil
+		return HandlerResult{Action: ActionReject}, fmt.Errorf("unable to process identity claim %s - invalid payload", msg.Header.ID)
```

Comment:
I feel like leaking in `fmt.Errorf` (rather than `i18n.NewError`) calls into the code is a bit of a slippery slope.
I understand we're making an assumption the caller won't return it anywhere... but the goal of the PR seemed to be that the raiser of an error wouldn't make assumptions about the caller.
Probably worth just popping these into `coremsg` - what do you think?

Reply:
I agree with this. Quite a few places in the code will simply continue to pass `err` back up until it is returned in an API response. I don't see any harm in returning a proper error code and translation even from lower level functions like this. If they get returned in an API response that's great. If they just get logged it's also great, and potentially in the operators native language.

Reply:
Yea, in the current world they're only logged, just because this happens on a background event loop that has no connection to a user action.

But for the forthcoming "gateway namespaces", much of this logic will be reused in a synchronous way, directly in response to a user action (such as when creating local identities and token pools). So in that case, we could bubble them up (instead of a more generic "definition rejected" error), and localizing them is probably the right call. It means ~30 new error strings.